content/english/security-phishing/1-Bob goes fishing/_index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Bob goes fishing"
```

Comment:
Please rename all the files to be of the following format, such that none of the pages are in their own folder.

content/english/security-phishing/bob-goes-fishing.md
content/english/security-phishing/phishing-basics.md

etc...